travis/setup_dependencies_common.sh

```diff
@@ -341,13 +344,13 @@ if [[ $NUMPY_VERSION == dev* ]]; then
     # We then install Numpy itself at the bottom of this script
     export CONDA_INSTALL="conda install $QUIET $PYTHON_OPTION $MKL"
 elif [[ $NUMPY_VERSION == stable ]]; then
-    retry_on_known_error conda install $QUIET --no-pin numpy=$LATEST_NUMPY_STABLE $MKL
     export NUMPY_OPTION="numpy=$LATEST_NUMPY_STABLE"
-    export CONDA_INSTALL="conda install $QUIET $PYTHON_OPTION numpy=$LATEST_NUMPY_STABLE $MKL"
+    export CONDA_INSTALL="conda install $QUIET $PYTHON_OPTION $NUMPY_OPTION $MKL"
+    NUMPY_INSTALL="conda install $QUIET --no-pin $PYTHON_OPTION $NUMPY_OPTION $MKL"
 elif [[ $NUMPY_VERSION == pre* ]]; then
-    retry_on_known_error conda install $QUIET --no-pin $MKL numpy
```

Comment:
here it doesn't matter, for prereleases numpy will be reinstalled anyway at the end.

Reply:
Sure, this will never fail. But there's no reason for not having the numpy installation in a single place either. IMHO this is cleaner.